views/membership.py

```diff
@@ -53,8 +57,13 @@ def create(self, request, *args, **kwargs):
         try:
             group = Group.objects.get(pk=group)
             if not has_admin_rights(person, group):
+                logger.warning(
+                    f'Person trying to create Membership does'
```

Comment:
No need to use `f''` if there is no interpolation happening. You should insert person IDs and other meaningful information into the error message.